docs/content/themes/unicorn.mdx

```diff
@@ -2,14 +2,7 @@
 title: Unicorn Theme
 ---
 
-import {
-  Box,
-  Column,
-  Columns,
-  Container,
-  Stack,
-  Text,
-} from '@marigold/components';
+import { Box, Columns, Container, Stack, Text } from '@marigold/components';
```

Comment:
Since a few weeks you can delete this line complete. We dont need marigold component imports anymore.